dist/css/style.css

```diff
@@ -1,10 +1,10 @@
 /*GENERAL
 /*TODO: Fix so that the body css is only rendered for the login page*/
-/*body {
+body {
   background-image: url(../images/background.jpg);
   background-position: center bottom;  
   background-size: cover;
-}*/
+}
 
```

Comment:
I'm okay with uncommenting this but if we do this it'll make the background image for all the pages, i.e the profile page and search page as well